### ISSUE TITLE
Describe command line args in documentation

### ISSUE THREAD

**Issue by kshitij86** [gkbrk/slowloris#41]
It'd be better if the documentation had more info than "can be used to modify the scripts behaviour".

**Comment by gkbrk** [comment]
@kshitij86 Looks like the project readme could use some touch-ups. Since you made an issue for it though, have your tried passing `-h` to the script for the help page? I think it clarifies most of the options.

```
usage: slowloris.py [-h] [-p PORT] [-s SOCKETS] [-v] [-ua] [-x] [--proxy-host PROXY_HOST] [--proxy-port PROXY_PORT] [--https]
                    [--sleeptime SLEEPTIME]
                    [host]

Slowloris, low bandwidth stress test tool for websites

positional arguments:
  host                  Host to perform stress test on

optional arguments:
  -h, --help            show this help message and exit
  -p PORT, --port PORT  Port of webserver, usually 80
  -s SOCKETS, --sockets SOCKETS
                        Number of sockets to use in the test
  -v, --verbose         Increases logging
  -ua, --randuseragents
                        Randomizes user-agents with each request
  -x, --useproxy        Use a SOCKS5 proxy for connecting
  --proxy-host PROXY_HOST
                        SOCKS5 proxy host
  --proxy-port PROXY_PORT
                        SOCKS5 proxy port
  --https               Use HTTPS for the requests
  --sleeptime SLEEPTIME
                        Time to sleep between each header sent.
```

A quick fix for the issue might be to just paste this in the readme.

**Comment by kshitij86** [comment]
Added this info in #42 .

**Comment by gkbrk** [comment]
Fixed with 421e85d4cd. Needs to be cleaned-up and improved later. Will also investigate a solution to auto-generate this from `slowloris -h` later.
